bindings/nodejs/tests/test_service.test.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { expect, test } from 'vitest'
+import { Operator } from '../index.js'
+
+const operator = new Operator('memory', { root: '/tmp' })
```

Comment:
We should not hardcode scheme here. Our test framework should access the scheme from input like `vitest memory` or `vitest s3`.

Reply:
The test suite could not received the custom args from the cli or `ctx` from the test framework. 

A tricky way is binding the scheme to the test file name, but I think it's worst than hardcode.

Reply:
Hi, thanks for the contribution, but this feature is required.

As you may have noticed, OpenDAL supports over 30 services, all of which need to be tested. Our developers will create local `.env` files, and the GitHub action will be set up using environment variables.

We expect to run tests such as `whatever s3` or `whatever fs`, so that we can maintain a consistent calling convention that benefits both the development experience and the automatic generation of GitHub actions.

Can we have different test classes which represent different services? Or we can switch to other frameworks?

Reply:
I changed another method maybe can achieve this requirement.